### PR TITLE
Bug 1774156: 4.2 cherry-picks in support of automated service ca rotation

### PR DESCRIPTION
**Pull request by marun**
~**NOTE** This PR is not mergeable until the updates to [library-go](https://github.com/openshift/library-go/pull/684) and [api](https://github.com/openshift/api/pull/577) have merged.~

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -15,11 +15,15 @@ import:
 - package: k8s.io/component-base
   version: kubernetes-1.14.0
 - package: github.com/openshift/api
-  version: master
+  # FAKE BUMP
+  repo: git@github.com:marun/api
+  version: 4.2-automated-rotation
 - package: github.com/openshift/client-go
-  version: master
```
Comment (deads2k) [inline glide.yaml]: phew, we have to do a better job :(

**Comment by deads2k** [comment]
/lgtm

don't release the hold until you really bump

**Comment by marun** [comment]
Bumped and ready for review.

**Comment by marun** [comment]
@sttts @stlaz merge plz?

**Pull request merged**